docs/user/dql/functions.rst

```diff
@@ -291,9 +291,19 @@ E
 Description
 -----------
 
-Specifications: 
+Usage: E() returns the Euler's number
 
-1. E() -> DOUBLE
+Return type: DOUBLE
+
+Example::
+
+    od> SELECT E()
+    fetched rows / total rows = 1/1
+    +-------------------+
+    | e()               |
```

Comment:
Sorry for last minute comment. Since we haven't added many functions yet, could you check if the column name required to be all upper case (ex. `E()`, `PI()`) or not? In MySQL or SQL standard.

Reply:
Good point! Checked it in MySQL, it should be consistent with the input cases

Reply:
Thanks for confirming. So the result column name is always lowercase for now? We can fix this separately if change is major.

Reply:
Yes all lowercase for now. I looked into this issue, it might be a major change. Created related issue: https://github.com/opendistro-for-elasticsearch/sql/issues/606

Reply:
Cool. Thanks!